### ISSUE TITLE
tappAS unable to connect to DB 

### ISSUE THREAD

**Issue by yueyaog** [ConesaLab/tappAS#44]
Hi! Thanks for creating this great tool to enable functional analysis for RNA-Seq data on isoform level. I am trying to use tappAS 1.0.7 to work with my Mas-Seq dataset. I followed your ISMB2020 tutorial. However, I can't create the project as I wanted. I checked the command line output.

```
createFolderRecursive: /Users/gaoyueya/tappasWorkspace/References/UserDefined.2145338620.tappas/
running...
added log txt(1): 10:18 - ERROR: Unable to connect to DB /Users/gaoyueya/tappasWorkspace/References/UserDefined.2145338620.tappas/annotations.db.

added log txt(1): 10:18 - java.sql.SQLException Code Exception: Error opening connection

stopped...
added log txt(1): 10:19 - Closing project 'Demo'
```
 It says I can't connect to the DB in that path. But I checked the path. Only one gff3 file was created.

```console
tappAS % ls /Users/gaoyueya/tappasWorkspace/References/UserDefined.2145338620.tappas/
annotations.gff3
```
My guess is tappAS trying to create `annotations.db`, but instead it created `annotations.gff3`. Could anyone help me with this issue. Thank you.

**Comment by yueyaog** [comment]
Hi! I wasn't using my own data in the above comment. I used all the materials that was provided in ISMB2020 tutorial.

**Comment by psalguerog** [comment]
Hello @yueyaog ,

Could you contact me by email or by the tappAS in the next [link](https://app.tappas.org/contact/)?

I would like to share a beta version of the new release of tappAS to see if you have the same problem there.

Regards,
Pedro Salguero

**Comment by yueyaog** [comment]
Hi @psalguerog, I just sent you an email using the tappAS link. Thanks!

**Comment by psalguerog** [comment]
I'm sorry to hear that. And also, I would like to say sorry again because,
until January, I will not be able to check the problem.

But as the problem is not happening to any other else, try to delete de
tappasWorkspace folder to generate it again. Inside that folder, all the
annotations files are kept between versions (1.0.7 and 1.0.8), so maybe the
problem is transferred between versions. If not working yet, try to
reinstall Java. tappAS can work with Java 7/8.

Best regards,
Pedro Salguero

**Comment by wendy0853** [comment]
I am getting the same issue of  ERROR: Unable to connect to DB when I'm trying to run the Demo that was instructed in the installation file. I tried deleting tappasWorkspace folder or reinstalling Java and none of those options worked.

tappas version: 1.0.8
ProcessID: 1773
JavaFX: ver 8.0.441-b07
OS: Mac OS X ver 14.5
Architecture: aarch64
Available processors, cores, for JVM: 11
Available memory, for JVM: 8.00 GB
Rscript version: N/A
